bw2io/extractors/simapro_lcia_csv.py

```diff
@@ -123,6 +198,30 @@ def read_metadata(cls, data, index):
 
     @classmethod
     def read_method_data_set(cls, data, index, filepath):
+        """
+        Read method data set from `data` starting at `index`.
+
+        Parameters
+        ----------
+        data : list
+            A list of lists containing the data to be processed.
+        index : int
+            The starting index to read method data set from.
+        filepath : str
+            The file path of the method data set.
+
+        Returns
+        -------
+        list
+            A list of completed method data sets.
+        int
+            The index where the method data set reading ended.
+
+        Raises
+        ------
+        ValueError
```

Comment:
...previously you always mentioned _when_ the error was raised - why not here?

Reply:
Added. I missed this, meant to get back to it but forgot. Thanks for the catch.